vllm/attention/layer.py

```diff
@@ -48,6 +48,7 @@ def __init__(
         blocksparse_params: Optional[Dict[str, Any]] = None,
         logits_soft_cap: Optional[float] = None,
         per_layer_sliding_window: Optional[int] = None,
+        use_irope: bool = False,
```

Comment:
Do we need to add use_irope here?

Seems vllm/v1/attention/backends/flashinfer.py's change should be enough?